docs/requirements.txt

```diff
@@ -1,8 +1,7 @@
-docutils>=0.17.1
 furo>=2022.6.21
-jinja2<3.1.0
-myst-nb>=0.16.0
-myst-parser>=0.17.0
+jinja2
```

Comment:
@John-P please can you confirm why it was pinned to less than 3.1.0?

Reply:
This was pinned because of an incompatibility with another library. I can't remember which but it should have been included in a comment somewhere.

Reply:
I believe it was the current version of flask to work. This may have been fixed on the latest flaks but I have not tested this.

Reply:
ok I will leave this PR for now.

Reply:
Looks like it might be fixed on sphinx. I see their requirements has >=3.0: https://github.com/sphinx-doc/sphinx/blob/cd3f2e435000835dd98a11497440bc16a79ec31c/pyproject.toml#L64

Also, readthedocs also automatically installs [sphinx-jinja2-compat](https://pypi.org/project/sphinx-jinja2-compat/) during our builds.

It must be an issue for local builds still. Did you get it to build on your setup with new jinja2 (>3.1)?

Reply:
Local build also installs [sphinx-jinja2-compat](https://pypi.org/project/sphinx-jinja2-compat/) and ignores latest version of jinja2 in favour of 3.0.3.